src/GapicClientTrait.php

```diff
@@ -81,6 +81,7 @@ trait GapicClientTrait
         Call::SERVER_STREAMING_CALL => 'startServerStreamingCall',
     ];
     private bool $isNewClient;
+    private $newClient;
```

Comment:
Just a doubt, where does this get instantiated? Or any setter for this in this trait?

Reply:
Woops, this is a typo. But as you can see, above this is the variable `$isNewClient`, so what we want is to use `$isNewClient` where we use `$newClient`.